### ISSUE TITLE
Evaluateion technics

### ISSUE THREAD

**Issue by cathyghotbi** [cathyghotbi/drug_discovery_ML#4]
**Acceptance Criteria:**
- Gather information about different evaluation metrics in drug discovery,
- Decide which metric fits BBBP

**Description:**
Further steps:
Drug discovery data is often imbalanced.
Useful metrics:
- ROC-AUC
- Precision / Recall
- Confusion matrix

`from sklearn.metrics import roc_auc_score
preds = model.predict_proba(X_test)[:,1]
auc = roc_auc_score(y_test, preds)
print("ROC-AUC:", auc)`

**Comment by cathyghotbi** [comment]
investigation information gather in below wiki:
https://github.com/cathyghotbi/drug_discovery_ML/wiki/Evalucation-metrics-in-drug-discovery

selected metrics: ROC-AUC — ranking quality and Confusion matrix — classification errors
